azure-nextgen-ts-appservice/index.ts

```diff
@@ -0,0 +1,153 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as insights from "@pulumi/azure-nextgen/insights/latest";
+import * as resource from "@pulumi/azure-nextgen/resources/latest";
+import * as sql from "@pulumi/azure-nextgen/sql/latest";
+import * as storage from "@pulumi/azure-nextgen/storage/latest";
+import * as web from "@pulumi/azure-nextgen/web/latest";
+import * as pulumi from "@pulumi/pulumi";
+
+// use first 10 characters of the stackname as prefix for resource names
+const prefix = (pulumi.getProject()+pulumi.getStack()).substring(0, 9);
```

Comment:
Any particular reason to do this? I'm happy to remove that, similar to other examples.